src/templates/nationals/2018/contact.html

```diff
@@ -13,18 +13,23 @@
   </div>
   <p>
     <div id="faq" role="tablist">
```

Comment:
Oops, there's only supposed to be one of these divs, and it's supposed to contain all of the faq_items.  Would you mind also deleting the other </div><div id="faq"...>s?

Reply:
Thanks!